pkg/s3store/s3store.go

```diff
@@ -724,9 +727,7 @@ func (upload *s3Upload) concatUsingDownload(ctx context.Context, partialUploads
 	if err != nil {
 		return err
 	}
-	fmt.Println(file.Name())
```

Comment:
This looked like debugging/testing residue, so I removed it. Let me know if that's not the case.

Reply:
Yes, you are right! Thanks!